packages/framework/package.json

```diff
@@ -154,6 +154,7 @@
   "dependencies": {
     "@novu/shared": "workspace:*",
     "ajv": "^8.12.0",
+    "ajv-errors": "^3.0.0",
```

Comment:
added `ajv-errors` for the custom error messages on JsonSchema, check example below